### PR TITLE
Make page gap configurable

### LINKED ISSUE #2315: Make the scroll mode's page gap height configurable
I haven't found a setting in `defaults.lua` that would allow me to configure the height of the horizontal gray band that appear between pages in scroll mode.

I would like to set it to *0*.

(I'm using build *666*.)

### PR DESCRIPTION
**Pull request by baskerville**
Fixes #2315.

### CODE REVIEW

**Comment by Frenzie** [comment]
This does seem sufficiently niche that a GUI setting would be overkill. Anyhoo, I'll wait for Travis to see if this breaks any tests (you never know =P).

File: frontend/apps/reader/modules/readerview.lua [hunk 1]
```diff
@@ -43,9 +43,9 @@ local ReaderView = OverlapGroup:new{
     page_states = {},
     scroll_mode = "vertical",
     page_gap = {
-        width = Screen:scaleBySize(8),
-        height = Screen:scaleBySize(8),
-        color = Blitbuffer.COLOR_GREY,
+        width = Screen:scaleBySize(DPAGE_GAP_WIDTH),
```
Comment (houqp) [inline frontend/apps/reader/modules/readerview.lua]: Could we do this with `G_reader_settings:readSetting` instead? We are in the processing of migrating everything in global DEFAULTS to `reader.settings.lua` to reduce number of global variables. Then you can you move the comments into the code. I am thinking of writing a custom parser to parse all these tweakable settings to automatically generate a user manual.

**Comment by houqp** [comment]
Sorry, should have mentioned to you in the original issue that we are ditching defaults.lua for performance and maintainability reasons :-/

**Comment by baskerville** [comment]
Done.

**Comment by houqp** [comment]
Thanks for the patch!

**Pull request merged**